python_modules/dagster/dagster/_core/definitions/op_invocation.py

```diff
@@ -81,13 +87,16 @@ def _separate_args_and_kwargs(
             resource_inputs_from_args_and_kwargs[resource_arg] = kwargs[resource_arg]
 
     adjusted_kwargs = {
-        k: v for k, v in kwargs.items() if k not in resource_inputs_from_args_and_kwargs
+        k: v
+        for k, v in kwargs.items()
+        if k not in resource_inputs_from_args_and_kwargs and k != "config"
```

Comment:
with our new understanding of the definition or the word resource I think we can safely rename this to `resources_from_args_and_kwargs`. Alternatively `resources_by_param_name`